twitter_api/main.py

```diff
@@ -16,6 +21,158 @@ def before_request():
 
 
 # implement your views here
+def query_db(query, args=(), one=False):
+    cur = g.db.execute(query, args)
+    rv = [dict((cur.description[idx][0], value)
+               for idx, value in enumerate(row)) for row in cur.fetchall()]
```

Comment:
is this a for loop within a for loop? if so, might want to see whether you can do it a faster way (cos right now it runs in n^2 time)